scripts/cpplint.py

```diff
@@ -283,7 +280,6 @@
     'build/include_order',
     'build/include_what_you_use',
     'build/namespaces_headers',
-    'build/namespaces_literals',
```

Comment:
Could you tell me why choose to remove `'build/namespaces_literals` amd `build/c++14` and `build/c++tr1`?

Reply:
I removed `build/namespaces_literals`, `build/c++14`, and `build/c++tr1` because there are no more lint checks in `cpplint.py` that produce warnings in those categories and all mentions of those categories in "NOLINT" comments have been removed from the source code. So leaving them around is kind of like leaving "dead code", IMO.